src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
```

Comment:
With the new docs site, we are using a newer version of MDX that doesn't require the spacing in between the JSX tags. Markdown will now be properly processed if you write this like a normal JSX component. This should now work (unlike the dev site)

```js
<Callout variant="tip">
  Some **markdown** inside the `Callout` component.
</Callout>
```

Reply:
Thanks @jerelmiller! I'll remove these unnecessary lines!